test/androidTest/java/org/thoughtcrime/securesms/espressoTests/ConversationChatNameTest.java

```diff
@@ -0,0 +1,58 @@
+package org.thoughtcrime.securesms.espressoTests;
+
+import android.Manifest;
+import android.support.test.espresso.ViewInteraction;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.rule.GrantPermissionRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.test.suitebuilder.annotation.LargeTest;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.thoughtcrime.securesms.ConversationListActivity;
+import org.thoughtcrime.securesms.R;
+import org.thoughtcrime.securesms.util.ConversationActions;
+import org.thoughtcrime.securesms.util.GenericActions;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+
+@LargeTest
+@RunWith(AndroidJUnit4.class)
+public class ConversationChatNameTest {
+
+    @Rule
+    public ActivityTestRule<ConversationListActivity> mActivityTestRule = new ActivityTestRule<>(ConversationListActivity.class);
+
+    @Test
+    public void conversationListActivityTestSettingChatName() {
+        ConversationActions.createNewConversation("5144022093");
```

Comment:
Are we sure we want to put this phone number up on git?

Reply:
I'll add a fake number (too short or too long)